pkg/lvm/lvm_util.go

```diff
@@ -357,9 +359,26 @@ func parseVolumeGroup(m map[string]string) (apis.VolumeGroup, error) {
 	return vg, nil
 }
 
+// ReloadLVMMetadataCache refreshes lvmetad daemon cache used for
+// serving vgs or other lvm utility.
+func ReloadLVMMetadataCache() error {
```

Comment:
@iyashu The reload will be performed everytime the syncHandler executes right? Will it have any impact when we reload so frequently?